functional-tests/setup.py

```diff
@@ -0,0 +1,37 @@
+import flexitest
```

Comment:
Oh god another file?
We already have `entry.py`, `factory.py`, `constants.py`, `utils.py`, `net_settings.py`, etc.

Can't we merge this into `entry.py`?

Also, `setup.py` is very ambiguous. Maybe a better name if we really need a new file.

Reply:
Merged into `entry.py`. These new classes will act as parent class for each test so i thought this is a setup for the tests. But then again alot of setup stuff is in entry.py